### PR TITLE
Feature/vqrc redesign

### PR DESCRIPTION
**Pull request by fedperezp**
### Why are we making these changes? Include references to any related Jira tasks or GitHub Issues

Jira Ticket:
https://venmoinc.atlassian.net/browse/VMORECOMBO-385
Elmo Experiment:
https://internal.msmaster.qa.paypal.com/elmo/experiment/119346

### Reproduction Steps (if applicable)
To alter the experiment, go to https://internal.msmaster.qa.paypal.com/elmo/experiment/119346 and set the guid to the variant of your choice.

### Screenshots (if applicable)

Captures taken before VMORECOMBO-386

| Default | Scanned | Authorized | Error | 
| ---- | ---- | ---- | ---- |
| image | image | image | image |
| image | image | image | image | image | image | image | image | image |

| Default A | Scanned A | Auth A | Error A |
| --- | --- | --- | --- |
| image | image | image | image |

From top to bottom: Ctrl, B variant (A variant + powered by paypal), A variant (light themed + details).

### Dependent Changes (if applicable)

Related PR adding experiment on SCNW:
https://github.paypal.com/Checkout-R/smartcomponentnodeweb/pull/219
Deploy Concerns:
None, we have the default version on PSPB even with missing callback from SCNW

❤️  Thank you!

### CODE REVIEW

File: src/button/pay.js [hunk 1]
```diff
@@ -43,6 +43,9 @@ export function getPaymentFlow({ props, payment, config, serviceData } : {| prop
 
     for (const flow of PAYMENT_FLOWS) {
         if (flow.isEligible({ props, config, serviceData }) && flow.isPaymentEligible({ props, payment, config, serviceData })) {
+            // return flow;
+        }
```
Comment (fedperezp) [inline src/button/pay.js:47]: Use this for local testing of the vqrc - I'll revert it after reviews

File: server/components/qrcode/middleware.js [hunk 2]
```diff
@@ -39,13 +47,14 @@ export function getQRCodeMiddleware({ logger = defaultLogger, cache, cdn = !isLo
                     // width:  240,
                     margin: 0,
                     color:  {
-                        dark:  VENMO_BLUE,
+                        dark:  '#008CFF',
```
Comment (gabrielo91) [inline server/components/qrcode/middleware.js:50]: Could you keep this value in a variable in constants?

File: src/qrcode/components.jsx [hunk 3]
```diff
@@ -217,6 +308,11 @@ export const cardStyle : string = `
         font-size: 20px;
         color: #2F3033;
     }
+    #view-boxes.${ VQRC_VARIANT.LIGHT } #fee-disclaimer {
+        font-size: 36px;
+        line-height: 40px;
+        padding: 0;
```
Comment (gabrielo91) [inline src/qrcode/components.jsx:314]: isn't 0 the default value for padding?

File: server/components/qrcode/middleware.js (hunk 2 above)
Comment (fedperezp) [reply server/components/qrcode/middleware.js:50]: sure

**Review by TNuzzi — APPROVED** [review]

**Pull request merged**